docs/ota-client-guide/modules/ROOT/pages/add-board-class.adoc

```diff
@@ -0,0 +1,45 @@
+= Add a board class for new target board in meta-updater
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+Adding a board class for a new target board in meta-updater involves two main steps:
+
+. adding a new `sota_\{MACHINE}.bbclass` file for the board
+
++
+Using an NXP board as an example, we created the bbclass `classes/sota_ls1043ardb.bbclass` and added the following to the class file:
++
+[source,bash]
+----
+KERNEL_IMAGETYPES = "fitImage" <1>
+
+OSTREE_KERNEL = "fitImage-${INITRAMFS_IMAGE}-${MACHINE}-${MACHINE}"
+OSTREE_KERNEL_ARGS = "console=ttyS0,115200 ramdisk_size=8192 root=/dev/ram0 rw rootfstype=ext4 ostree_root=/dev/mmcblk0p2"
+
+WKS_FILE_sota = "ls1043ardb-ota.wks" <2>
+IMAGE_BOOT_FILES = "ls1043ardb_boot.scr" <3>
+----
++
+<1> For most boards, especially boards that use a device tree, we recommend you use a https://elinux.org/images/f/f4/Elc2013_Fernandes.pdf[FIT (flattened image tree)] image if possible. A FIT image includes all the components of an initial boot image--like the device tree, initramfs, and kernel--in a bundle.

```

Comment:
At least in github, the numbering style is different here (lower case Roman numerals). If Antora renders it correctly, that's fine, but there might be something wonky here.